src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const space = document.querySelector('ul');
+
+function conversion(value) {
+  return +value.dataset.salary
+    .split('').slice(1).join('').split(',').join('');
+}
+
+function sortList1(list) {
```

Comment:
The 'sortList1' function name does not properly describe its purpose. Consider renaming it to 'sortListBySalaryDescending'.